src/main/java/com/commercetools/sync/products/ProductSyncOptions.java

```diff
@@ -27,22 +27,22 @@
     // optional filter which can be applied on generated list of update actions
     private Function<List<UpdateAction<Product>>, List<UpdateAction<Product>>> filterActions;
 
-    public ProductSyncOptions(@Nonnull final CtpClient ctpClient,
-                              @Nonnull final BiConsumer<String, Throwable> errorCallBack,
-                              @Nonnull final Consumer<String> warningCallBack,
-                              final boolean removeOtherLocales,
-                              final boolean removeOtherSetEntries,
-                              final boolean removeOtherCollectionEntries,
-                              final boolean removeOtherProperties,
-                              final boolean compareStaged,
-                              final boolean publish,
-                              final boolean removeOtherVariants,
-                              @Nonnull final List<String> whiteList,
-                              @Nonnull final List<String> blackList,
-                              @Nonnull final Function<List<UpdateAction<Product>>
-                                      , List<UpdateAction<Product>>> filterActions) {
+    ProductSyncOptions(@Nonnull final CtpClient ctpClient,
```

Comment:
That is why there is [https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/products/ProductSyncOptions.java#L14-L17](this) comment on top of it :)